<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBFieldEdit</class>
 <widget class="QDialog" name="DBFieldEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBField Editor</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="enterLabel">
         <property name="text">
          <string>Enter Name</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fieldName">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="fieldLabel">
         <property name="text">
          <string>DBField label</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLineEdit" name="fieldValue">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="fieldTxtEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="fieldTable">
         <column>
          <property name="text">
           <string>values</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="tableEntries">
         <property name="maximum">
          <number>99999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>DBField type</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeBox"/>
       </item>
       <item>
        <widget class="QCheckBox" name="hexBox">
         <property name="text">
          <string>show int in hex</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer14">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>151</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
